server/controllers/FollowController.js

```diff
@@ -0,0 +1,201 @@
+import models from '../database/models';
+import userProfile from '../utils/helpers/profileHelper';
+
+const { User, Follow } = models;
+
+/**
+ *
+ * The FollowController contains static methods that are used as
+ * controllers to handle the different Follow routes.
+ *  @class FollowController
+ *  @returns {undefined} this is a class thus does not return anything
+ */
+class FollowController {
+  /**
+   * add a user to the follower list of an user
+   * @param {Object} req the request body
+   * @param {Object} res the response body
+   * @param {function} next a call to the next function
+   * @returns {Object} the response body
+   */
+  static follow(req, res, next) {
+    const { id: followerId } = req.body.decoded;
+    const followingId = parseInt(req.params.userId, 10);
+
+    // // Ensure a user cannot unfollow him/her self
```

Comment:
You should use just a pair of `/` for inline comments. Also, I think you might mean `follow` instead of `unfollow`.